include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -479,19 +572,19 @@ private :
         //segment on equator
         //Note: antipodal points on equator does not define segment on equator
         //but pass by the pole
-        CT diff = geometry::math::longitude_distance_signed<geometry::radian>(lon1, lon2);
+        CT diff = geometry::math::longitude_distance_signed<geometry
+            ::radian>(lon1, lon2);
```

Comment:
This line break is interesting. Are you using some kind of automatic fomatter?

Reply:
could be from the previous IDE, cutting the minimal part that exceeds 80 lines.
I put it in one line now